pages/api/elements.py

```diff
@@ -290,6 +291,18 @@ class PageElementDetailAPIView(TrailMixin, generics.RetrieveAPIView):
     def get_object(self):
         return self.element
 
+    def get(self, request, *args, **kwargs):
+        try:
```

Comment:
Let's just add a comment here that states "Marking the last time the PageElement was read by a user following it." otherwise it takes some time to understand why this is done.